docs/cristin-proxy-swagger.yaml

```diff
@@ -2570,6 +2615,11 @@ components:
         institutionsResponsibleForResearch:
           - type: 'Organization'
             id: 'https://api.dev.nva.aws.unit.no/cristin/organization/215.0.0.0'
+        healthProjectData:
```

Comment:
Duplications. Define examples and refer to them.

Reply:
Can examples be reused ? Do you have a code block i can use as starting point?

Reply:
I don't think you can use $ref in examples, unfortunately